oam/install_scripts/syslogSetup.sh

```diff
@@ -180,23 +191,23 @@ if [ ! -z "$syslog_conf" ] ; then
 		      group=`stat -c "%U %G" /var/log/syslog | awk '{print $2}'`
 		fi
```

Comment:
Here we seem to be redefining user and group to be the u/g of whoever owns the system log file...?